src/app/store/provider.tsx

```diff
@@ -0,0 +1,8 @@
+'use client';
+
+import { Provider } from 'react-redux';
+import { store } from './store';
+
+export function Providers({ children }) {
```

Comment:
1) Call it `Provider` - still it is only 1 provider
2) Use `PropsWithChildren` for type `({children}: PropsWithChildren)`
3) Convert this function to an arrow function. By default, it is better to use arrow functions. it will show you are familiar with modern JS syntax